### PR TITLE
Added smAllice

### PR DESCRIPTION
**Pull request by Armastardo**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/smallice/config.h [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x514b // QK Quark
```
Comment (drashna) [inline keyboards/smallice/config.h:23]:
```suggestion
#define VENDOR_ID    0x514B // QK Quark
```

File: keyboards/smallice/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+![smAllice](https://i.imgur.com/HAiJnnK.png)
+
+A 40% alice-like keyboard with an arrow cluster.
+
+* Keyboard Maintainer: [Brandon Beltran](https://github.com/armastardo)
+* Hardware Supported: smAllice PCB
+* Hardware Availability: [KeyHive](http://keyhive.xyz/)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/smallice/readme.md:11]: Please add instructions on how to jump into the bootloader, to flash new firmware.

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/smallice/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        KC_ESC,  KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,     KC_P,   KC_LBRC, KC_RBRC, KC_BSPC, \
+        KC_VOLU, KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,     KC_SCLN,KC_QUOT,          KC_ENT,  \
+        KC_VOLD, KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_B,    KC_N,    KC_M,    KC_COMM,  KC_DOT, KC_RSFT, KC_UP,   KC_SLSH, \
+                 KC_LCTRL,         KC_LALT,          KC_SPC,  KC_LWIN,          KC_SPC,           MO(1),            KC_LEFT, KC_DOWN, KC_RIGHT \
+    ),
+    [1] = LAYOUT(
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+                 _______,          _______,          _______, _______,          _______,          _______,          _______, _______, _______  \
```
Comment (fauxpark) [inline keyboards/smallice/keymaps/default/keymap.c:30]:
```suggestion
        KC_ESC,  KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,     KC_P,   KC_LBRC, KC_RBRC, KC_BSPC,
        KC_VOLU, KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,     KC_SCLN,KC_QUOT,          KC_ENT,
        KC_VOLD, KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_B,    KC_N,    KC_M,    KC_COMM,  KC_DOT, KC_RSFT, KC_UP,   KC_SLSH,
                 KC_LCTRL,         KC_LALT,          KC_SPC,  KC_LWIN,          KC_SPC,           MO(1),            KC_LEFT, KC_DOWN, KC_RIGHT
    ),
    [1] = LAYOUT(
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
                 _______,          _______,          _______, _______,          _______,          _______,          _______, _______, _______
```

File: keyboards/smallice/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+                 _______,          _______,          _______, _______,          _______,          _______,          _______, _______, _______  \
+    ),
+    [3] = LAYOUT(
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+                 _______,          _______,          _______, _______,          _______,          _______,          _______, _______, _______  \
```
Comment (fauxpark) [inline keyboards/smallice/keymaps/via/keymap.c:42]:
```suggestion
        KC_ESC,  KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,     KC_P,   KC_LBRC, KC_RBRC, KC_BSPC,
        KC_VOLU, KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,     KC_SCLN,KC_QUOT,          KC_ENT,
        KC_VOLD, KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_B,    KC_N,    KC_M,    KC_COMM,  KC_DOT, KC_RSFT, KC_UP,   KC_SLSH,
                 KC_LCTRL,         KC_LALT,          KC_SPC,  KC_LWIN,          KC_SPC,           MO(1),            KC_LEFT, KC_DOWN, KC_RIGHT
    ),
    [1] = LAYOUT(
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
                 _______,          _______,          _______, _______,          _______,          _______,          _______, _______, _______
    ),
    [2] = LAYOUT(
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
                 _______,          _______,          _______, _______,          _______,          _______,          _______, _______, _______
    ),
    [3] = LAYOUT(
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
                 _______,          _______,          _______, _______,          _______,          _______,          _______, _______, _______
```

File: keyboards/smallice/keymaps/via/readme.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# VIA keymap for smAllice
+
+![smAllice](https://i.imgur.com/OngGf9n.png))
```
Comment (fauxpark) [inline keyboards/smallice/keymaps/via/readme.md:3]:
```suggestion
![smAllice](https://i.imgur.com/OngGf9n.png)
```

File: keyboards/smallice/smallice.json [hunk 6]
```diff
@@ -0,0 +1,226 @@
+{
```
Comment (fauxpark) [inline keyboards/smallice/smallice.json:1]: This file belongs in the VIA repo, not here.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/smallice/readme.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+    make smallice:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+### Flash
+
+- Plug in
+- Press reset button
+- Flash using QMK Toolbox or dfu-util (`make smallice:<keymap>:dfu-util`)
```
Comment (noroadsleft) [inline keyboards/smallice/readme.md:21]:
```suggestion
### Flash

- Plug in
- Press reset button
- Flash using QMK Toolbox or dfu-util (`make smallice:<keymap>:flash`)

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**